### PR TITLE
Fix toggle selection

### LINKED ISSUE #1427: Show if table or tile is selected
**Wanna have the bug fixed quickly?**
Visit Issue hunt...
[Issue hunt](https://issuehunt.io/r/FAForever/downlords-faf-client)

### PR DESCRIPTION
**Pull request by 1-alex98**
Fix #1427

### CODE REVIEW

**Review by Katharsas — CHANGES_REQUESTED** [review]
This breaks my client layout slightly:
Without PR:
css_tiles_1
With PR:
css_tiles_2

You may need to set Windows Scaling to 125 or 115 percent to reproduce, not sure.

**Comment by 1-alex98** [comment]
This makes not much sense tho because all i did was chnage background colour and spacing

File: src/main/resources/theme/play/custom_games.fxml [hunk 1]
```diff
@@ -86,7 +87,7 @@
                     </content>
                 </ScrollPane>
                 <AnchorPane fx:id="gameViewContainer" maxHeight="1.7976931348623157E308"
-                            maxWidth="1.7976931348623157E308" GridPane.rowIndex="1" prefWidth="10000"/>
+                            maxWidth="1.7976931348623157E308" prefWidth="10000" GridPane.rowIndex="1"/>
```
Comment (1-alex98) [inline src/main/resources/theme/play/custom_games.fxml:90]: Maybe this causes issue tho i did not set this this was scene builder...

Comment (Katharsas) [reply src/main/resources/theme/play/custom_games.fxml:90]: I don't see the difference, do you mean the order?

**Comment by Katharsas** [comment]
Let me check again against the base between this branch and develop instead of develop. Maybe some other change on develop causes this.

File: src/main/resources/theme/play/custom_games.fxml (hunk 1 above)
Comment (1-alex98) [reply src/main/resources/theme/play/custom_games.fxml:90]: oh no i thought it added prefWidth

**Comment by Katharsas** [comment]
No its definitely this commit causing the layout change. Probably because the additional highlighting adds some space here and there which throws of the rest of the layout.
Edit: Oh wait.
I think i know whats happening.

**Comment by Katharsas** [comment]
The problem is horizontal resizing. If you make the width of the client smaller and smaller, at some point the tile settings will be split into two lines. This looks horrible in all versions of the client (as in the screen). More than 2 lines is basically unusable.
Your change adds some horizontal space to the toggle buttons, which means that now when i am in fullscreen, the tile options won't fit into one line (as you can see in the screenshot, the sorting option flowed to the second line), whereas before it did barely fit in one line.

So i think its just too many options without enough horizontal space which is made slightly worse by this change.

**Comment by 1-alex98** [comment]
Well if i dont add the spacing it looks awful believe me

**Comment by Katharsas** [comment]
Its really a different issue i think (the multi line layout), so im fine with this. Havent looked at the code yet.

**Comment by Katharsas** [comment]
I want to test this again, can you rebase?

**Comment by Katharsas** [comment]
The weird thing is that in the table view elements resize perfectly. Only in the tiles view do they not.

**Comment by Katharsas** [comment]
Ok found i pushed a fix for the resize problem.

**Review by Katharsas — APPROVED** [review]

**Pull request merged**